model_builder/model_builder/handlers/project.py

```diff
@@ -27,10 +27,10 @@
 # Note APIHandler.finish() sets Content-Type handler to 'application/json'
 # Use finish() for json, write() for text
 
-from .util import StochSSFolder, StochSSProject, StochSSModel, StochSSSpatialModel, \
-                  StochSSAPIError, report_error, report_critical_error
+from .util import GillesPy3DFolder, GillesPy3DProject, GillesPy3DModel, GillesPy3DSpatialModel, \
+                  GillesPy3DAPIError, report_error, report_critical_error
 
-log = logging.getLogger('stochss')
+log = logging.getLogger('model_builder')
```

Comment:
Change `log = logging.getLogger('model_builder')` to `log = logging.getLogger('gillespy3d')`